### PR TITLE
[INGEST-76] Updating to librdkafka 1.8.2

### PR DESCRIPTION
**Pull request by darena-zendesk**
## Why?

My team (@zendesk/ingest) maintains the support views pipeline that uses Racecar. Our consumers hang often and never recover. As a first step, we would like to upgrade Racecar to use the latest for rdkafka-ruby (0.11.0) which upgrades  librdkafka to 1.8.2. Librdkafka 1.8.0 change log contains changes in this area.

### References:

- Librdkafka 1.8.0 release notes: https://github.com/edenhill/librdkafka/blob/cfc0731617ddd6858058f31f564772202f209e72/CHANGELOG.md#librdkafka-v180
- Librdkafka 1.8.2 release notes: https://github.com/edenhill/librdkafka/blob/cfc0731617ddd6858058f31f564772202f209e72/CHANGELOG.md#librdkafka-v182
- rdkafka-ruby PR upgrading to librdkafka 1.8.2: https://github.com/appsignal/rdkafka-ruby/pull/180

## How?

I made the following changes:

- Updated `racecar.gemspec` to require rdkafka 0.11.0
- Committed the changes to `Gemfile.lock` that were generated by running `bin/setup`

### Testing

I followed the instructions for setting up the Racecar development environment in the README and confirmed (dockerized) tests pass locally by running `docker-compose run --rm tests rspec`.

### Considerations

- I noticed that `spec/integration/consumer_spec.rb` tests only passed when ran in isolation. I observed this on both `master` and my branch.

Thanks in advance for your time! 🙇‍♀️

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["2.4", "2.5", "2.6", "3.0"]
+        ruby-version: ["2.6", "3.0"]
```
Comment (dasch) [inline .github/workflows/ci.yml:15]: Why this change? That's breaking any existing client on an older version of Ruby, so we'd need a reason, and an entry in the changelog.

**Review by dasch — CHANGES_REQUESTED** [review]
I'm :+1: if we can either avoid the bump to the required Ruby version or have a good reason for it.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (darena-zendesk) [reply .github/workflows/ci.yml:15]: i need to add into the change log that rdkafka 0.11.0 requires ruby 2.6+ (https://rubygems.org/gems/rdkafka/versions/0.11.0). i was trying to get CI to pass, so i updated that collection, but couldn't find the definition that ensure the project passed on ruby 2.5.

Comment (dasch) [reply .github/workflows/ci.yml:15]: :+1: can you also rebase on latest master?

**Comment by leonmaia** [comment]
Hey @darena-zendesk this is just a gentle bump  on this PR  😅, could you rebase this with master? This change is very helpful and I know of a couple teams including mine that would love to have this merged as soon as possible 😄.

**Comment by leonmaia** [comment]
Merged on #283
